lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +20,19 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if type(text) != str:
+        return None
+
+    text = text.lower()
+
+    letters_list = []
+
+    for i in range(len(text)):
```

Comment:
do you really need to iterate over length?

Reply:
fixed